drake/solvers/CMakeLists.txt

```diff
@@ -32,28 +32,23 @@ if(MATLAB_FOUND)
   target_link_libraries(lqrmex drakeUtil)
 endif()
 
+set(optimization_files)
+if (snopt_c_FOUND)
+  list(APPEND optimization_files SNOPTWrapper.cpp)
+else()
+  list(APPEND optimization_files NoSNOPT.cpp)
+endif(snopt_c_FOUND)
+include_directories(${PROJECT_SOURCE_DIR}/core)
```

Comment:
Shouldn't do this anymore! Going to mess things up for projects that only use the Drake installed headers.